mrc/localization/camera/reader.py

```diff
@@ -24,8 +24,11 @@ def __init__(self, resolution=(640, 480)):
         self.camera = PiCamera()
         self.camera_resolution = (math.ceil(resolution[1] / 16) * 16, math.ceil(resolution[0] / 32) * 32, 3)
         self.camera.resolution = resolution
-        self.camera.brightness = 30
-
+        self.camera.brightness = brightness
+        self.camera.contrast = contrast
+        self.camera.sharpness = sharpness
+        self.camera.saturation = saturation
+        self.camera.exposure_mode = 'night'
```

Comment:
Check if needed

Reply:
I'd leave it because it was used when trying different parameters.